app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -107,9 +107,29 @@ def textual_endpoints
       :value => @record.connection_configurations.hawkular.endpoint.port}]
   end
 
+  def redact_passwords_from_proxy_values(name, value)
+    unless ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job::PROXY_ENV_VARIABLES.include?(name)
```

Comment:
@mzazrivec How would I access the decorator? Also, the decorators are for either kubernetes or Openshift, is it possible to have a parent one for container managers that they will inherit from?